index.html

```diff
@@ -600,6 +629,16 @@
         <h2>
           <code>MediaTrackConstraintSet</code> dictionary
         </h2>
+        <p>
+          <a>ConstraintSet</a> dictionary specifies each member's set of
```

Comment:
<a>MediaTrackConstraintSet</a> partial dictionary extends the original <a>MediaTrackConstraintSet</a>. It specifies each member's set of allowed values.

I would keep the original sentence too here as it is sufficient to explain.